datamodel/changelogs/0001/03_tww_db_dss.sql

```diff
@@ -119,29 +119,29 @@ FOR EACH ROW EXECUTE PROCEDURE
 
 -------
 -------
-CREATE TABLE tww_od.progression_alternative
+CREATE TABLE tww_od.txt_progression_alternative
```

Comment:
Is the `txt_` prefix right here? Because further down by the constraints and values is not updated